web/src/css/author.module.css

```diff
@@ -0,0 +1,7 @@
+.container{
+    background-color: white;
+}
+
+.text{
+    color:black;
```

Comment:
Do you really need another class for the text color? It doesn't work putting it inside `.container` style?

Reply:
[fix] remove .text class and write its content in .container class